<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="MyWidget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="connectBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>261</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="regionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>region_name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="queueUrlLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>queue_url</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLineEdit" name="region_Input">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="queueUrl_Input">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="close_Btn">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>-1</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>remember</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>region_Input</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>setConnectBtnEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queueUrl_Input</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>setConnectBtnEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>setRemember()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setConnectBtnEnable()</slot>
  <slot>setRemember()</slot>
 </slots>
</ui>
